splice/web/api/campaign.py

```diff
@@ -89,7 +89,11 @@ def get(self):
 
         Takes an optional account_id as a query string argument.
         """
-        campaigns = get_campaigns(request.args.get('account_id'))
+        parser = reqparse.RequestParser()
```

Comment:
I just created a self.reqparse_get for this. Totally up to you, not an issue at all.